Templates/XCTest/APIConfigTesting.swift

```diff
@@ -0,0 +1,66 @@
+//  Copyright © 2019 The Bow Authors.
+
+import Foundation
+
+public extension API.Config {
+    
+    /// Stubs data to respond the next HTTP request with.
+    /// - Parameter data: Contents of the response to the next request.
+    /// - Parameter code: HTTP status code for the response. Defaults to 200.
+    func stub(data: Data, code: Int = 200) -> API.Config {
+        StubURL.reset()
```

Comment:
This is nice at the moment and for this PR, but we could consider a more general approach where you can stub multiple results to test an orchestration of different calls and stubbing a particular endpoint. Something like:

```swift
stub("/user/profile", data: Data(), code: 200)
stub("/user/images", data: Data(), code: 200)
```

What do you think?